src/templates/base/2019/base.html

```diff
@@ -278,6 +289,81 @@
           });
         }
 
+
+        var languageYearNavList = document.querySelectorAll('.nav-dropdown-list');
+        for (let i = 0; i < languageYearNavList.length; i++) {
+          const list = languageYearNavList[i];
+          
+        }
+
```

Comment:
Does this do anything now? Can we remove?

```suggestion
```

Reply:
Good catch, thanks, was meant to be removed